patches/minecraft/net/minecraft/client/gui/GuiScreen.java.patch

```diff
@@ -35,15 +35,20 @@
          {
              GlStateManager.func_179101_C();
              RenderHelper.func_74518_a();
-@@ -418,6 +428,7 @@
+@@ -414,10 +421,12 @@
+ 
+     public void func_175281_b(String p_175281_1_, boolean p_175281_2_)
+     {
++        if((p_175281_1_=net.minecraftforge.event.ForgeEventFactory.onClientSendMessage(p_175281_1_)).isEmpty())return;
```

Comment:
No... This code is like `ImmutableList.builder().add('a').add('b');`;
let's have a vote, :+1: for agreeing with me, :-1: for agreeing with @killjoy1221.